src/hostapi/wasapi/pa_win_wasapi.c

```diff
@@ -2458,7 +2290,7 @@ PaError PaWasapi_Initialize( PaUtilHostApiRepresentation **hostApi, PaHostApiInd
 
 #ifndef PA_WINRT
     // Fail safely for any Windows version below Windows Vista
```

Comment:
This comment now seems incorrect, since paOsVersionWindowsUnknown no longer means "version below Vista"

Not sure whether it's the code or the comment that needs to change.

Reply:
Hi Ross! Thank you for noticing this, made correction to detect OS below Vista.